Improve SITL instructions (#533)
betaflight/betaflight.com: docs/development/SITL.md

Run `make TARGET=SITL`

> Start Betaflight: `./obj/main/betaflight_SITL.elf`
> 

Reviewer: This is already described in Start and Run
```suggestion
```

./websockify 127.0.0.1:6761 127.0.0.1:5761
```

Open the Betaflight Online Configurator, and in Options enable "manual connection mode"

> Then establish a connection by putting address `ws://127.0.0.1:6761` in the Port field

Reviewer 2: ```suggestion
Then establish a connection by using address `ws://127.0.0.1:6761` in the Port field and clicking Connect.
```
i prefer "using" over "putting". The `and clicking Connect.` is optional, or may be written differently.

### Start and Run

> 1. Start Betaflight in terminal: `./obj/main/betaflight_SITL.elf`

Reviewer: ```suggestion
1. Start Betaflight: `./obj/main/betaflight_SITL.elf`
```

> 1. Start Betaflight if it's not already running: `./obj/main/betaflight_SITL.elf`

Reviewer: Documentation should be concise
```suggestion
1. Start Betaflight `./obj/main/betaflight_SITL.elf`
```
